metals/src/main/scala/scala/meta/internal/pantsbuild/commands/BloopGlobalSettings.scala

```diff
@@ -49,4 +63,29 @@ object BloopGlobalSettings {
     }
 
   }
+
+  case class Property(metalsProperty: String) {
```

Comment:
Can we move this to the `zipkin` package and make it a toplevel class? Might be good to also move the `optionValue` and `updateOption` methods to belong to `Property` somehow